src/api/api.cc

```diff
@@ -11105,21 +11109,35 @@ static const char* GetDisabledFeatureSpecifier() {
   return getenv("RECORD_REPLAY_DISABLE_FEATURES");
 }
 
+static bool GetTestEnvironmentFlag() {
+  auto* sTestEnvironment = getenv("RECORD_REPLAY_TEST_ENVIRONMENT");
+  return sTestEnvironment && !strcmp(sTestEnvironment, "1");
```

Comment:
This is a little weird, can we just check for the environment variable's existence like we do in other places?

Reply:
@bhackett1024 
I am generally opposed against that because the expectation is that `X=0` should disable, not enable, something.
But sure, if that's how we always do it, I'll go that way.

Reply:
Hmm, as a compromise could we use the TestEnv function from the backend's Utils.cpp?  That explicitly watches for "0" and but treats any other value for the env var as enabled.  It would be nice to standardize on this sort of stuff across our repos.

Reply:
I am copy-and-pasting the `Utils.cpp` solution here and also to `channel.ts` where its also parsed.
If we want to do this right, it would be great to have some shared utility library. But then, we still have to unify it with our TS `string-to-bool` conversion at the same time, so its a bit of effort that we might want to consider tackling in the near future 🤷